packages/block-library/src/embed/embed-no-preview.native.js

```diff
@@ -114,10 +118,10 @@ const EmbedNoPreview = ( { label, icon, isSelected, onPress } ) => {
 				onPress={ onPressContainer }
 			>
 				<View style={ containerStyle }>
-					<View style={ styles.embed__icon }>{ icon }</View>
+					<Icon icon={ icon } fill={ embedIconStyle.fill } />
```

Comment:
The `embed__icon ` style object defined the size of the icon, I'm wondering if we no longer need it 🤔 , is the default size the required one?

Reply:
Yeah it was 24px in there and it's the default on `Icon` component:

https://github.com/WordPress/gutenberg/blob/c3bb70882efe34f468c602ad8be47bba06a4d1da/packages/components/src/icon/index.tsx#L50-L54

Reply:
Thanks for the reference, in this case, looks great to me 🎊 !